pyscraper/regmem/commons/process.py

```diff
@@ -66,7 +69,7 @@ def recursive_fetch(
             "Take must be less than or equal to 20 - API limit annoyingly."
         )
 
-    bar = tqdm(desc="Fetching ", unit="items", disable=quiet)
+    bar = tqdm(desc="Fetching ", unit="items", disable=quiet or no_progress)
```

Comment:
As I understand it, this is the only actual change, and everything else is just passing the flag down from the top to the bottom. Feels like flags are a thing that can be global but perhaps I'm too old school ;-)